2019-09-22 15:02:58.727 [error] <0.271.0> CRASH REPORT Process <0.271.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module Pipeline was given as a child to a supervisor but it does not exist.">>} in 'Elixir.Supervisor':init_child/1 line 629 in application_master:init/4 line 138
2019-09-22 19:15:16.062 [error] <0.279.0> gen_server 'Elixir.Pipeline.Broadway.Producer_default_1' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.368.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.373.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 19:15:16.062 [error] <0.280.0> gen_server 'Elixir.Pipeline.Broadway.Producer_default_2' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.369.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.372.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 19:15:16.069 [error] <0.279.0> CRASH REPORT Process 'Elixir.Pipeline.Broadway.Producer_default_1' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.368.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.373.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 19:15:16.069 [error] <0.280.0> CRASH REPORT Process 'Elixir.Pipeline.Broadway.Producer_default_2' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.369.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.372.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 19:15:16.070 [error] <0.278.0> Supervisor 'Elixir.Pipeline.Broadway.ProducerSupervisor' had child 'Elixir.Pipeline.Broadway.Producer_default_1' started with 'Elixir.Broadway.Producer':start_link([{module,{'Elixir.BroadwayRabbitMQ.Producer',[{queue,<<"ampq.queue">>},{connection,[{username,...},...]}]}},...], [{name,'Elixir.Pipeline.Broadway.Producer_default_1'}]) at <0.279.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.368.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.373.0>},60000]}} in context child_terminated
2019-09-22 19:15:16.070 [error] <0.278.0> Supervisor 'Elixir.Pipeline.Broadway.ProducerSupervisor' had child 'Elixir.Pipeline.Broadway.Producer_default_2' started with 'Elixir.Broadway.Producer':start_link([{module,{'Elixir.BroadwayRabbitMQ.Producer',[{queue,<<"ampq.queue">>},{connection,[{username,...},...]}]}},...], [{name,'Elixir.Pipeline.Broadway.Producer_default_2'}]) at <0.280.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.369.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.372.0>},60000]}} in context child_terminated
2019-09-22 20:09:13.002 [error] <0.280.0> gen_server 'Elixir.Pipeline.Broadway.Producer_default_2' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.369.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.373.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:09:13.002 [error] <0.279.0> gen_server 'Elixir.Pipeline.Broadway.Producer_default_1' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.368.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.372.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:09:13.017 [error] <0.280.0> CRASH REPORT Process 'Elixir.Pipeline.Broadway.Producer_default_2' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.369.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.373.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:09:13.017 [error] <0.278.0> Supervisor 'Elixir.Pipeline.Broadway.ProducerSupervisor' had child 'Elixir.Pipeline.Broadway.Producer_default_2' started with 'Elixir.Broadway.Producer':start_link([{module,{'Elixir.BroadwayRabbitMQ.Producer',[{queue,<<"ampq.queue">>},{connection,[{username,...},...]}]}},...], [{name,'Elixir.Pipeline.Broadway.Producer_default_2'}]) at <0.280.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.369.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.373.0>},60000]}} in context child_terminated
2019-09-22 20:09:13.018 [error] <0.279.0> CRASH REPORT Process 'Elixir.Pipeline.Broadway.Producer_default_1' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.368.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.372.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:09:13.018 [error] <0.278.0> Supervisor 'Elixir.Pipeline.Broadway.ProducerSupervisor' had child 'Elixir.Pipeline.Broadway.Producer_default_1' started with 'Elixir.Broadway.Producer':start_link([{module,{'Elixir.BroadwayRabbitMQ.Producer',[{queue,<<"ampq.queue">>},{connection,[{username,...},...]}]}},...], [{name,'Elixir.Pipeline.Broadway.Producer_default_1'}]) at <0.279.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.368.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.372.0>},60000]}} in context child_terminated
2019-09-22 20:09:13.026 [error] <0.374.0> gen_server 'Elixir.Pipeline.Broadway.Producer_default_2' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.398.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.402.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:09:13.026 [error] <0.375.0> gen_server 'Elixir.Pipeline.Broadway.Producer_default_1' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.400.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.403.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:09:13.027 [error] <0.374.0> CRASH REPORT Process 'Elixir.Pipeline.Broadway.Producer_default_2' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.398.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.402.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:09:13.027 [error] <0.278.0> Supervisor 'Elixir.Pipeline.Broadway.ProducerSupervisor' had child 'Elixir.Pipeline.Broadway.Producer_default_2' started with 'Elixir.Broadway.Producer':start_link([{module,{'Elixir.BroadwayRabbitMQ.Producer',[{queue,<<"ampq.queue">>},{connection,[{username,...},...]}]}},...], [{name,'Elixir.Pipeline.Broadway.Producer_default_2'}]) at <0.374.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.398.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.402.0>},60000]}} in context child_terminated
2019-09-22 20:09:13.027 [error] <0.375.0> CRASH REPORT Process 'Elixir.Pipeline.Broadway.Producer_default_1' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.400.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.403.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:09:13.027 [error] <0.278.0> Supervisor 'Elixir.Pipeline.Broadway.ProducerSupervisor' had child 'Elixir.Pipeline.Broadway.Producer_default_1' started with 'Elixir.Broadway.Producer':start_link([{module,{'Elixir.BroadwayRabbitMQ.Producer',[{queue,<<"ampq.queue">>},{connection,[{username,...},...]}]}},...], [{name,'Elixir.Pipeline.Broadway.Producer_default_1'}]) at <0.375.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.400.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.403.0>},60000]}} in context child_terminated
2019-09-22 20:11:42.050 [error] <0.365.0> Supervisor {<0.365.0>,amqp_channel_sup} had child channel started with amqp_channel:start_link(network, <0.346.0>, 1, <0.367.0>, {<<"client 127.0.0.1:49659 -> 127.0.0.1:5672">>,1}) at <0.368.0> exit with reason noproc in context shutdown_error
2019-09-22 20:11:42.051 [error] <0.364.0> Supervisor {<0.364.0>,amqp_channel_sup} had child channel started with amqp_channel:start_link(network, <0.349.0>, 1, <0.366.0>, {<<"client 127.0.0.1:49658 -> 127.0.0.1:5672">>,1}) at <0.369.0> exit with reason noproc in context shutdown_error
2019-09-22 20:11:52.367 [error] <0.280.0> gen_server 'Elixir.Pipeline.Broadway.Producer_default_2' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.404.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.406.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:11:52.377 [error] <0.280.0> CRASH REPORT Process 'Elixir.Pipeline.Broadway.Producer_default_2' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.404.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.406.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:11:52.378 [error] <0.278.0> Supervisor 'Elixir.Pipeline.Broadway.ProducerSupervisor' had child 'Elixir.Pipeline.Broadway.Producer_default_2' started with 'Elixir.Broadway.Producer':start_link([{module,{'Elixir.BroadwayRabbitMQ.Producer',[{queue,<<"ampq.queue">>},{connection,[{username,...},...]}]}},...], [{name,'Elixir.Pipeline.Broadway.Producer_default_2'}]) at <0.280.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.404.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.406.0>},60000]}} in context child_terminated
2019-09-22 20:11:52.385 [error] <0.407.0> gen_server 'Elixir.Pipeline.Broadway.Producer_default_2' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.419.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.421.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:11:52.386 [error] <0.407.0> CRASH REPORT Process 'Elixir.Pipeline.Broadway.Producer_default_2' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.419.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.421.0>},60000]}} in gen_server:call/3 line 223
2019-09-22 20:11:52.386 [error] <0.278.0> Supervisor 'Elixir.Pipeline.Broadway.ProducerSupervisor' had child 'Elixir.Pipeline.Broadway.Producer_default_2' started with 'Elixir.Broadway.Producer':start_link([{module,{'Elixir.BroadwayRabbitMQ.Producer',[{queue,<<"ampq.queue">>},{connection,[{username,...},...]}]}},...], [{name,'Elixir.Pipeline.Broadway.Producer_default_2'}]) at <0.407.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'ampq.queue' in vhost '/'">>}},{gen_server,call,[<0.419.0>,{subscribe,{'basic.consume',0,<<"ampq.queue">>,<<>>,false,false,false,false,[]},<0.421.0>},60000]}} in context child_terminated
2019-09-22 23:07:15.006 [error] <0.358.0> gen_server <0.358.0> terminated with reason: {socket_error,timeout}
2019-09-22 23:07:15.006 [error] <0.349.0> gen_server <0.349.0> terminated with reason: {socket_error,timeout}
2019-09-22 23:07:15.023 [error] <0.359.0> gen_server <0.359.0> terminated with reason: {socket_error,timeout}
2019-09-22 23:07:15.024 [error] <0.346.0> gen_server <0.346.0> terminated with reason: {socket_error,timeout}
2019-09-22 23:07:15.187 [error] <0.349.0> CRASH REPORT Process <0.349.0> with 0 neighbours exited with reason: {socket_error,timeout} in gen_server:handle_common_reply/8 line 751
2019-09-22 23:07:15.187 [error] <0.347.0> Supervisor {<0.347.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.348.0>, {amqp_params_network,<<"guest">>,<<"guest">>,<<"/">>,"localhost",5672,0,0,10,50000,none,[fun amqp_a..>,...],...}) at <0.349.0> exit with reason {socket_error,timeout} in context child_terminated
2019-09-22 23:07:15.188 [error] <0.347.0> Supervisor {<0.347.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.348.0>, {amqp_params_network,<<"guest">>,<<"guest">>,<<"/">>,"localhost",5672,0,0,10,50000,none,[fun amqp_a..>,...],...}) at <0.349.0> exit with reason reached_max_restart_intensity in context shutdown
2019-09-22 23:07:15.188 [error] <0.346.0> CRASH REPORT Process <0.346.0> with 0 neighbours exited with reason: {socket_error,timeout} in gen_server:handle_common_reply/8 line 751
2019-09-22 23:07:15.188 [error] <0.334.0> Supervisor {<0.334.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.337.0>, {amqp_params_network,<<"guest">>,<<"guest">>,<<"/">>,"localhost",5672,0,0,10,50000,none,[fun amqp_a..>,...],...}) at <0.346.0> exit with reason {socket_error,timeout} in context child_terminated
2019-09-22 23:07:15.188 [error] <0.334.0> Supervisor {<0.334.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.337.0>, {amqp_params_network,<<"guest">>,<<"guest">>,<<"/">>,"localhost",5672,0,0,10,50000,none,[fun amqp_a..>,...],...}) at <0.346.0> exit with reason reached_max_restart_intensity in context shutdown
